examples/kubernetes/addons/prometheus/prometheus.yaml

```diff
@@ -1,171 +0,0 @@
----
```

Comment:
I believe this file is being used in our tutorials

Reply:
Fixed, update docs.